.swift-version

```diff
@@ -1 +1 @@
-3.0
+4.0
```

Comment:
Same version as R. swift

I fixed warning, and
I checked compile (swift 4.0 and 5.0) with this visibility.